### DISCUSSION TITLE
Allow Saved Posts, Recent Mentions Cross-Team

### DISCUSSION THREAD

**Discussion by matthewbirtch** [mattermost/community#43] [Enhancements]
Because Saved Posts and Recent Mentions are more personal to me, it would be better if they were cross-team so I don't have to switch teams to find the right posts

**Comment by bbodenmiller** [comment]
This would be awesome. Same goes for drafts - would be good if I could see all my drafts across all teams in one place.

**Reply by esethna** [reply]
I opened a new discussion for this one! https://github.com/mattermost/community/discussions/63

**Comment by esethna** [comment]
Similarly extending this to search! (recent mentions + saved posts are basically search afaik).

Challenge with that one ^ is performance on servers with many teams
